lisa/operating_system.py

```diff
@@ -382,6 +383,16 @@ def capture_system_information(self, saved_path: Path) -> None:
             saved_path / "modinfo-hv_netvsc.txt"
         )
 
+        if hasattr(self._node, "first_initialize") and self._node.first_initialize:
+            from lisa.tools import SystemdAnalyze
+
+            try:
+                systemd_analyze_tool = self._node.tools[SystemdAnalyze]
+                boot_time = systemd_analyze_tool.get_boot_time()
+                notifier.notify(boot_time)
```

Comment:
@squirrelsc I am not sure how to pass the testresults id into provision boot time message, there is testresultid column in provisionboottime table.

Reply:
The boot message is not related to test cases. So we may consider remove test result id from the message.

Reply:
then we need add platform, kernelversion, hostversion (if it has), image, vmsize, location info for provisionbootmessage.